src/test/java/core/basesyntax/FruitTransactionTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitTransactionTest {
```

Comment:
For the future, you should test only logic in Services, Perositories, Handlers
Test cases are rarely written for models

Reply:
Thanksm now I know it)